tasks/system_probe.py

```diff
@@ -493,13 +503,38 @@ def build_network_ebpf_link_file(ctx, parallel_build, build_dir, p, debug, netwo
             asynchronous=parallel_build,
         )
 
+def build_http_ebpf_files(ctx, build_dir):
+    network_bpf_dir = os.path.join(".", "pkg", "network", "ebpf")
+    network_c_dir = os.path.join(network_bpf_dir, "c")
+    network_prebuilt_dir = os.path.join(network_c_dir, "prebuilt")
+
+    uname_m = check_output("uname -m", shell=True).decode('utf-8').strip()
+    network_flags = get_ebpf_build_flags(target=["-target", "bpf"])
+    network_flags.append(f"-I{network_c_dir}")
+    network_flags.append(f"-D__{uname_m}__")
+
+    if uname_m == "aarch64":
+        network_flags.append("-isystem /usr/include/aarch64-linux-gnu")
```

Comment:
Why is this directory required? I'm worried it won't match the correct kernel version since it is out-of-tree from the rest of the headers.

Reply:
This is required for `asm/types.h`